packages/api/src/command/medical/admin/upload-doc.ts

```diff
@@ -32,34 +34,42 @@ export async function createAndUploadDocReference({
   };
 }): Promise<DocumentReference> {
   const patient = await getPatientOrFail({ id: patientId, cxId });
-  const organization = await getOrganizationOrFail({ cxId });
 
   const fhirApi = makeFhirApi(cxId);
-
-  const now = dayjs();
+  const refDate = dayjs();
+  const orgId = smallId();
+  const orgRef = `org${orgId}`;
+  const practitionerId = smallId();
+  const practitionerRef = `org${practitionerId}`;
```

Comment:
should this be `practitioner${practitionerId}` ?

Reply:
good catch

I had this fixed on this downstream PR: https://github.com/metriport/metriport/pull/890/files#diff-3d42b174b9456e6cba24c22730804f621f249c1013c2aedac4adc1c83c3635daR45